rpcs3/Emu/NP/np_handler.cpp

```diff
@@ -764,6 +769,7 @@ namespace np
 		memset(&npid, 0, sizeof(npid));
 		memset(&online_name, 0, sizeof(online_name));
 		memset(&avatar_url, 0, sizeof(avatar_url));
+		std::memset(&country_code, 0, sizeof(country_code));
```

Comment:
Any particular reason this is `std::memset` while the ones above are just `memset`?

Reply:
The idea was that you apply it to the others as well...